src/pages/Detail/Detail.js

```diff
@@ -17,14 +17,42 @@ function Detail() {
   const [product, setProduct] = useState([]);
   const [productName, productNameSet] = useState('');
   let productColor = [];
-  let productColorHex = [];
   let productSize = [];
-  const [productPrice, productPriceSet] = useState(1000);
+  const [productPrice, productPriceSet] = useState(100);
+
   let productImgUrl = [];
   const [imgUrl, urlSetting] = useState([]);
+
   const parsedQuery = queryString.parse(window.location.search);
   const getId = parsedQuery.productId;
   const [idValue, idSet] = useState(getId);
+
+  const [cartColor, cartColorChange] = useState('None');
+  const [cartSize, cartSizeChange] = useState('None');
+
+  const cartAdd = () => {
+    fetch(`http://localhost:8000/products/cart`, {
```

Comment:
```suggestion
    fetch(POST_CARTADD_API, {
```
config.js에 이름 편하신대로 수정해서 넣을면 좋을 것 같습니다.

Reply:
넵 감사합니다!